src/Routers/routes/ConsultationRoutes.tsx

```diff
@@ -22,6 +23,9 @@ export default {
   }: any) => (
     <ConsultationForm facilityId={facilityId} patientId={patientId} id={id} />
   ),
+  "/facility/:facilityId/patient/:patientId/consultation/:id/edit": () => (
+    <Error404 />
+  ),
```

Comment:
The intention of the issue is, that any invalid links should show a 404 page, not just `/edit`.

This can be achieved by having a fallback to Error404 component if none of the URLs has  matched with the current URL.